tests/unit/metadata/models/object.js

```diff
@@ -0,0 +1,78 @@
+import assert from 'assert';
+import ObjectMD from '../../../../lib/metadata/models/object/object';
+
+describe('Metadata regular object', () => {
+    let md = null;
+
+    beforeEach(() => {
+        md = new ObjectMD();
+    });
+
+    [
```

Comment:
Maybe add an explanatory comment:
`// In order: data property, value to set/get, default value`